packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 ## v2.1.0-dev (Jun 25, 2024)
 - Add `meta.displayName` to queries. It can be used to identify queries in performance metrics or logs. [#1895](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1895)
+- Upgrade to commerce-sdk-isomorphic v3.0.0 [#1914](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1914)
+
+### :warning: Planned API Changes :warning:
+#### Shopper Context
+
+Starting July 31st 2024, all endpoints in the Shopper context API will require the `siteId` parameter for new customers. This field is marked as optional for backward compatibility and will be changed to mandatory tentatively by January 2025. You can read more about the planned change [here](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-context?meta=Summary) in the notes section.
+
+#### Shopper Login (SLAS)
+
+SLAS will soon require new tenants to pass `channel_id` as an argument for retrieving guest access tokens. You can read more about the planned change [here](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas.html#guest-tokens).
+
+Please be aware that existing tenants are on a temporary allow list and will see no immediate disruption to service.  We do ask that all users seek to adhere to the `channel_id` requirement before the end of August to enhance your security posture before the holiday peak season.
+
+In practice, we recommend:
+- For customers using the SLAS helpers with a private client, it is recommended to upgrade to `v3.0.0` of the `commerce-sdk-react`.
```

Comment:
So the `commerce-sdk-react` consumes the new `commerce-sdk-isomorphic@3.0.0` which contains a breaking change where it now requires `channel_id` with SLAS private client. 

There were no breaking changes needed to consume this new version of the isomorphic SDK since we use siteId in the clientConfig by default, but PWA Kit customers who consume this new version may break depending on how they've modified their project. 

When we release the new version of `commerce-sdk-react` should it be `v3.0.0` or `v2.1.0`?

Reply:
It should be `commerce-sdk-react@3.0.0` since it includes a breaking change.